{{cookiecutter.project_slug}}/backend/app/pyproject.toml

```diff
@@ -30,18 +30,18 @@ dependencies = [
   "raven>=6.10.0",
   "jinja2>=3.1.2",
   "python-jose[cryptography]>=3.3.0",
-  "pydantic-settings==2.0.3",
+  "pydantic>=2.0",
+  "pydantic-settings>=2.0.3",
   "httpx>=0.23.1",
   "psycopg2-binary>=2.9.5",
   "setuptools>=65.6.3",
-  "sqlalchemy2-stubs>=0.0.2a29",
-  "beanie==1.22.6",
-  "motor==3.3.1",
+  "motor>=3.3.1",
   "pytest==7.4.2",
   "pytest-cov==4.1.0",
-  "pytest-asyncio==0.21.0",
+  "pytest-asyncio>=0.21.0",
   "argon2-cffi==23.1.0",
   "argon2-cffi-bindings==21.2.0",
+  "odmantic==1.0.0",
```

Comment:
I think these version pins should only apply to the major version, so `>=1.0,<2.0`.